egnoramoose.py

```diff
@@ -1,119 +1,167 @@
 class Pawn:
-	def __init__(self,location):
-		self.location = location
+    pass
+
+
+class Location:
```

Comment:
This class was created to handle the confusion with indexing the grid and handling user location input.